airgun/views/host.py

```diff
@@ -13,6 +14,9 @@
 from widgetastic.widget import Widget
 from widgetastic_patternfly import BreadCrumb
 from widgetastic_patternfly import Button
+from widgetastic_patternfly4.ouia import Button as pf4Button
```

Comment:
How about using `PF4Button` or `Pf4Buttton`?
```suggestion
from widgetastic_patternfly4.ouia import Button as PF4Button
```